lumigator/python/mzai/jobs/inference/tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+import json
+from pathlib import Path
+
+import pytest
+
+
+def load_json(path: Path) -> dict:
```

Comment:
do we need to load json directly from a file for these? or is it enough to have the direct structure, aka we could do:

```
@pytest.fixture(scope="session")  
def input_json():  
    return json.dumps(the minimal config here) 
```    
because we're not really testing the file itself but the actions on the json

Reply:
Yeah my reasoning was that examples might grow but I am ok to start with just a few dictionaries (rn it is 2 but I was already considering adding more, at least one for each model time)